terraform/gt-site.tf

```diff
@@ -0,0 +1,91 @@
+provider "aws" {
+  profile = "${var.profile}" # Assumes you have valid AWS credentials.
+  region  = "${var.region}"
+}
+
+# --- ECS RESOURCES --- #
+
+resource "aws_ecs_cluster" "gt-site-cluster" {
+  name = "gt-site-cluster"
+
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+# `containers.json` is analogous to `docker-compose.yml`.
+# All the container settings are there.
+resource "aws_ecs_task_definition" "gt-site-cluster-task" {
+  family                = "gt-site-cluster"
+  container_definitions = "${file("containers.json")}"
+}
+
+resource "aws_ecs_service" "gt-site-cluster-service" {
+  name                               = "gt-site-cluster"
+  cluster                            = "${aws_ecs_cluster.gt-site-cluster.id}"
+  task_definition                    = "${aws_ecs_task_definition.gt-site-cluster-task.family}:${aws_ecs_task_definition.gt-site-cluster-task.revision}"
+  desired_count                      = "${var.desired_instance_count}"
+  iam_role                           = "${var.ecs_service_role}"
```

Comment:
A concern is that changes made to the role to satisfy the copy-pasted project's purposes will automatically cascade to this one (either escalating privileges, or taking them away and causing breakage). In addition, `ecs_instance_profile` resolves to another Terraform managed resource.

Codifying the IAM resources used by each project in Terraform is preferred.